### DISCUSSION TITLE
Loading API types not recognizing post upgrade to 2.8.2

### DISCUSSION THREAD

**Discussion by brianmask** [quasarframework/quasar#14414] [General - Components / Directives / etc] [answered]
Hello,

I recently performed an upgrade of our quasar framework.
```
 @quasar/extras: 1.14.0 → 1.15.3
 quasar: 2.7.1 → 2.8.2
 @quasar/app-webpack: 3.5.3 → 3.6.0
 ```
 After the upgrade the QLoading API is no longer working under typescript giving the following error:
 ```
 ERROR in src/components/admin/EditField.vue:270:41
TS2339: Property 'show' does not exist on type '(opts?: QLoadingShowOptions | undefined) => (props?: any) => void'.
    268 |
    269 |     const setLoading = () => {
  > 270 |       if (pageLoading.value) $q.loading.show();
        |                                         ^^^^
    271 |       else $q.loading.hide();
    272 |     };
    273 |

ERROR in src/components/admin/EditField.vue:271:23
TS2339: Property 'hide' does not exist on type '(opts?: QLoadingShowOptions | undefined) => (props?: any) => void'.
    269 |     const setLoading = () => {
    270 |       if (pageLoading.value) $q.loading.show();
  > 271 |       else $q.loading.hide();
        |                       ^^^^
    272 |     };
```
All other type checking appears to be working as expected with the extras installed : 'Loading', 'LocalStorage', 'Notify'

I've deleted the node_modeuls folder, and reinstalled through yarn with no change.  Everything was working as expected prior to the upgrade.  Reverting the package.json / yarn.lock and reinstalling gets the types working again for the Loading API.  

I'm at a loss as to what is causing this one type not to function as designed.  

I'm sure it has something to do with typescript and not quasar specifically, however the upgrade performed recently is consistently the catalyst in all tests.

**Comment by yusufkandemir** [comment, marked as answer]
Fixed in #14395, will be included in the next release.

**Reply by brianmask** [reply]
Thank you @yusufkandemir glad I didn't spend too much time trying to figure out how I broke my typescript.  Appreciate the response.
